README.md

```diff
@@ -22,7 +22,8 @@ yarn dev
 1. Checkout a branch prefixed with either `feature/your-branch`, `fix/your-brach`, or `hotfix/your-branch` (depending on what type of PR it is)
 1. Make your awesome changes
 1. Make sure your changes are rebased on top of master (`git pull origin master`, `git rebase master`)
-1. Test it out by running `yarn dev` again, just to make sure :) **Since we don't have any tests yet.. this is important.**
+1. Test it out by running `yarn dev` again, just to make sure :)
+1. Run a visual [BackstopJS](https://github.com/garris/BackstopJS) test by running `npm run vtest` (you must run `yarn dev` before visual testing). Check that your changes look correct.
```

Comment:
We should change this to `yarn vtest` instead of `npm run vtest` since we use yarn here. Should do the same thing :)